projects/plugins/jetpack/modules/custom-css/custom-css.php

```diff
@@ -1646,208 +1646,6 @@ static function filter_attr( $css, $element = 'div' ) {
 	}
 }
 
-function migrate() {
```

Comment:
This function is called in the following plugin:

- [ ] `reflect` reflect.php,93 and 142

My guess is this is a badly named function on our part and they're declaring their own version.

Reply:
[This plugin defined a function of the same name](https://plugins.trac.wordpress.org/browser/reflect/trunk/reflect.php#L113). False positive.